src/styles/themes/global.css

```diff
@@ -0,0 +1,8 @@
+/** 
+* Currently used to hide images based on theme
+* see components/themed-image
+ */
```

Comment:
This is a new file where we can place global theme related config. I didn't want to put it in the 'dark' theme css file since this should be independent of the theme state.